oauth2/handler.go

```diff
@@ -42,8 +44,21 @@ type Handler struct {
 
 	AccessTokenLifespan time.Duration
 	CookieStore         sessions.Store
-
+	
 	L logrus.FieldLogger
+
+	Issuer string
+}
+
+// swagger:model WellKnown
+type WellKnown struct {
+	Issuer        string   `json:"issuer"`
```

Comment:
Nice! Thanks for adding the model here :) You could copy the descriptions from the JWK RFC here so it's clear what each of those values are.